migrations/20200406155335_map_prototcol_fate.js

```diff
@@ -0,0 +1,67 @@
+const transform = version => {
+  if (!version) {
+    return;
+  }
+  if (!version.protocols || !Array.isArray(version.protocols)) {
+    return version;
+  }
+  version.protocols.forEach(protocol => {
+    if (!protocol || !protocol['killing-method'] || !Array.isArray(protocol['killing-method'])) {
+      return;
+    }
+    if (protocol['killing-method'].includes('other')) {
+      return protocol['non-schedule-1'] = true;
+    }
+    if (protocol['killing-method'].includes('schedule-1')) {
+      return protocol['non-schedule-1'] = false;
+    }
+
+  });
```

Comment:
It's possibly worth setting a flag to check if any data has been changed here, so if no changes are required we can return undefined and avoid a db write with no changes.

Reply:
good idea